pkg/app/piped/cloudprovider/lambda/client.go

```diff
@@ -74,10 +74,9 @@ func (c *client) Apply(ctx context.Context, fm FunctionManifest, role string) er
 		Code: &lambda.FunctionCode{
 			ImageUri: aws.String(fm.Spec.ImageURI),
 		},
+		PackageType:  aws.String("Image"),
```

Comment:
I thought about other use-cases when our users may want to be able to use other kinds of PackageType ( Zip or S3 object ). But currently, we only support container image packaging, so let me hard code this PackageType value here as is, and update this later when we have a plan to support a wide range of package types.

Reply:
This is an internal change so I am fine about it.
👍